rotkehlchen/db/dbhandler.py

```diff
@@ -208,16 +208,17 @@ def __init__(
         self.conn_transient: DBConnection = None  # type: ignore
         # Lock to make sure that 2 callers of get_or_create_evm_token do not go in at the same time
         self.get_or_create_evm_token_lock = Semaphore()
-        self._connect(password)
-        self._check_unfinished_upgrades(password)
-        self._run_actions_after_first_connection(password)
+        self.password = password
```

Comment:
you should also unset this when logging out. Otherwise we have password remnants in memory which is bad